### DISCUSSION TITLE
所有视频打开都是哔哩哔哩漫画直播间

### DISCUSSION THREAD

**Discussion by jhs114514** [ywmoyue/biliuwp-lite#595] [bug]
### Bug 描述

所有视频打开都是哔哩哔哩漫画直播间，貌似线路错乱了，这种情况是偶尔能复现的

### 操作步骤

```plain text
1、打开应用
2、点击视频
```

### 期望结果

希望点开视频能播放对应的视频

### 截图

https://github.com/ywmoyue/biliuwp-lite/assets/149226254/5067b29a-1d5e-4b69-81ee-2c465d2999e1

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

win10 22h2

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
请发送日志，这个问题应该是处理视频链接时优先匹配到直播的正则了

**Reply by AvalonUltra** [reply]
附日志文件
[20240425.log](https://github.com/ywmoyue/biliuwp-lite/files/15119524/20240425.log)

**Comment by AvalonUltra** [comment]
+1
